Sources/_StringProcessing/Algorithms/Algorithms/Contains.swift

```diff
@@ -46,6 +46,20 @@ extension BidirectionalCollection where Element: Comparable {
   }
 }
 
+// Overload breakers
+
+extension StringProtocol {
+  @available(SwiftStdlib 5.7, *)
+  public func contains(_ other: String) -> Bool {
+    firstRange(of: other) != nil
+  }
+  
+  @available(SwiftStdlib 5.7, *)
+  public func contains(_ other: Substring) -> Bool {
+    firstRange(of: other) != nil
+  }
+}
```

Comment:
Do we need to add these to the proposal?

Reply:
I don't think so - we just need to make sure these are winning the overload resolution vs Foundation's overlay version. However we do that is just an implementation detail.